install_config/install/host_preparation.adoc

```diff
@@ -319,6 +319,22 @@ devices, which is not supported for production use and only appropriate for
 proof of concept environments. For production environments, you must create a
 thin pool logical volume and re-configure Docker to use that volume.
 
+[NOTE]
```

Comment:
you might want to consider linking to this section:  https://docs.openshift.org/latest/scaling_performance/optimizing_storage.html#choosing-a-graph-driver